tm2/pkg/std/tx.go

```diff
@@ -136,3 +144,36 @@ func (fee Fee) Bytes() []byte {
 	}
 	return bz
 }
+
+func LoadTxs(ctx context.Context, reader io.Reader) ([]Tx, error) {
```

Comment:
same here, I think `ParseTxs` should be a better name

Reply:
done